test/cpp/jit/backend_with_compiler.cpp

```diff
@@ -0,0 +1,111 @@
+#include <torch/csrc/jit/backends/backend.h>
+
+namespace torch {
+namespace jit {
+
+namespace {
+std::vector<std::string> parseBlob(const std::string& blob) {
+  std::vector<std::string> result;
+  std::stringstream s_stream(blob);
+  while (s_stream.good()) {
+    std::string substr;
+    getline(s_stream, substr, ',');
+    result.push_back(substr);
+  }
+  return result;
+}
+} // namespace
+// This test JIT backend is intended to do the minimal amount of work
+// necessary to test that the JIT backend registration endpoints and
+// code generation are working correctly. It is not intended to
+// produce numerically correct results.
+class BackendWithCompiler : public PyTorchBackendInterface {
+ public:
+  // Constructor.
+  explicit BackendWithCompiler() {}
+  virtual ~BackendWithCompiler() = default;
+
+  c10::impl::GenericDict compile(
+      c10::IValue processed,
+      c10::impl::GenericDict method_compile_spec) override {
+    return processed.toGenericDict();
```

Comment:
This is the next TODO right? Actually receive +1 methods and split them here.

Reply:
```processed``` here is a dict with method names as keys. So it's already split? I was thinking of to move the parsing part here: to parse to blob to something that can be directly used in execute.

Reply:
You're right, that makes sense.